cli/internal/state/state.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+// package state defines the structure of the Constellation state file.
+package state
+
+// State describe the entire state to describe a Constellation cluster.
+type State struct {
+	Version        string         `yaml:"version"`
+	Infrastructure Infrastructure `yaml:"infrastructure"`
+}
+
+// Infrastructure describe the state related to the cloud resources of the cluster.
+type Infrastructure struct {
+	UID               string   `yaml:"uid"`
+	PublicIP          string   `yaml:"publicIP"`
+	InitSecret        string   `yaml:"initSecret"`
```

Comment:
I'm unsure if `InitSecret` contains potentially unprintable characters.
It was of type `[]byte` in the id file, maybe we should keep it as such here?
@3u13r can you give an opinion on this?

Reply:
Going for `string` is fine here. The `initSecret` is generated in terraform (e.g. https://github.com/edgelesssys/constellation/blob/main/cli/internal/terraform/terraform/gcp/main.tf#L69) and in terraform `random_password` is equivalent to `random_string` which only consists of printable characters.